playground/mocks/data/idp/idx/terminal-return-email-consent.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
SIW gets this terminal view remediation when the user clicks on `Yes, it's me` button on the `consent` screen.